backend/ttnn_visualizer/views.py

```diff
@@ -395,6 +395,34 @@ def get_profiler_performance_data(session: Instance):
         result = csv.get_all_entries(as_dict=True, limit=100)
         return jsonify(result)
 
+@api.route("/profiler/perf-results/list", methods=["POST"])
+def get_profiler_performance_data_list():
+    location = request.json.get("location")
+    if location == "remote":
+        report_directory = Path(current_app.config["REMOTE_DATA_DIRECTORY"])
+    else:
+        report_directory = Path(current_app.config["LOCAL_DATA_DIRECTORY"])
+
+    path = f"{report_directory}/profiles"
```

Comment:
Since you are using Pathlib, the Pythonic way with a Path option would be to use the `/` operator. Then you wouldn't have to instantiate another Path object below with `Path(path)`.

```suggestion
    path = report_directory / "profiles"
    folder_names = [folder.name for folder in path.iterdir() if folder.is_dir()]
```

Reply:
Changed.